pkg/golinters/usetesting/testdata/usetesting_go124.go

```diff
@@ -0,0 +1,41 @@
+//go:build go1.24
+
+//golangcitest:args -Eusetesting
+package testdata
+
+import (
+	"context"
+	"os"
+	"testing"
+)
+
+func Test_contextBackground(t *testing.T) {
+	context.Background() // want `context\.Background\(\) could be replaced by t\.Context\(\) in .+`
+}
+
+func Test_contextTODO(t *testing.T) {
+	context.TODO() // want `context\.TODO\(\) could be replaced by t\.Context\(\) in .+`
+}
+
+func Test_osChdir(t *testing.T) {
+	os.Chdir("") // want `os\.Chdir\(\) could be replaced by t\.Chdir\(\) in .+`
+}
+
+func Test_osMkdirTemp(t *testing.T) {
+	os.MkdirTemp("", "") // want `os\.MkdirTemp\(\) could be replaced by t\.TempDir\(\) in .+`
+}
+
+func Test_osSetenv(t *testing.T) {
+	os.Setenv("", "")
+}
```

Comment:
Do you need the last 3, here?

Reply:
I need it to be sure that no element other than the Go version impacts the analysis.